src/class-wpml-gutenberg-strings-registration.php

```diff
@@ -52,24 +52,29 @@ public function register_strings( WP_Post $post, $package_data ) {
 		$this->leftover_strings = $original_strings = $this->string_translation->get_package_strings( $package_data );
 		$this->string_location  = 1;
 
+		$stringCleanUp = apply_filters( 'wpml_pb_get_string_clean_up', '', $post->ID );
```

Comment:
@brucepearson, please add a PHPDoc for this new filter hook. It's not obvious which return type we are expecting.

Reply:
@strategio I think I might change this and remove the filter and call a function directly. In reality $stringCleanUp will always be a StringCleanUp instance.

Reply:
@strategio I refactored this to remove the dependency on `StringCleanUp`

Reply:
Well done @brucepearson :)